venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,362 @@
+                "required": False
+                },
+            "title": {
+                "description": "Title of paper. Add TeX formulas using the following formats: $In-line Formula$ or $$Block Formula$$",
+                "order": 10,
+                "value-regex": ".{1,250}",
+                "required": True
+            },
+            "authors": {
+                "description": "Comma separated list of author names.",
+                "order": 11,
+                "values-regex": "[^;,\\n]+(,[^,\\n]+)*",
+                "required": True,
+                "hidden": True
+            },
+            "authorids": {
+                "description": "Search author profile by first, middle and last name or email address. If the profile is not found, you can add the author by completing first, middle, and last names as well as author email address.",
+                "order": 12,
+                "values-regex": "~.*|([a-z0-9_\\-\\.]{1,}@[a-z0-9_\\-\\.]{2,}\\.[a-z]{2,},){0,}([a-z0-9_\\-\\.]{1,}@[a-z0-9_\\-\\.]{2,}\\.[a-z]{2,})",
+                "required": True
+            },
+            "abstract": {
+                "description": "Abstract of paper. Add TeX formulas using the following formats: $In-line Formula$ or $$Block Formula$$",
+                "order": 14,
+                "value-regex": "[\\S\\s]{1,5000}",
+                "required": True
+            },
+            "pdf": {
+                "description": "Upload a single PDF containing the paper, references and any appendices",
+                "order": 15,
+                "value-file": {
+                "fileTypes": [
+                "pdf"
+                ],
+                "size": 80
+                },
+                "required": True
+            },
+        "software": {
+                "description": "Each ARR submission can be accompanied by one .tgz or .zip archive containing software (max. 200MB).",
+                "order": 21,
+                "value-file": {
+                "fileTypes": [
+                "tgz",
+                "zip"
+                ],
+                "size": 200
+                },
+                "required": False
+                },
+            "data": {
+                "description": "Each ARR submission can be accompanied by one .tgz or .zip archive containing data (max. 200MB).",
+                "order": 22,
+                "value-file": {
+                "fileTypes": [
+                "tgz",
+                "zip"
+                ],
+                "size": 200
+                },
+                "required": False
+                }        
+        }
+    }
+)
+
+client.post_invitation(submission_invitation)
+
+# Posting Blind Submission Invitation 
+blind = openreview.Invitation(
+    id = 'aclweb.org/ACL/2022/Conference/-/Blind_Submission',
+    readers = ['everyone'],
+    writers = ['aclweb.org/ACL/2022/Conference'],
+    invitees = ['~', 'OPenReview.net/Support'],
```

Comment:
```suggestion
    invitees = ['aclweb.org/ACL/2022/Conference'],
```

venue id should be enough